dark/views/tournament/tround/info.py

```diff
@@ -22,9 +26,17 @@ def post(self, request, *args, **kwargs):
         get_kwargs = {
             self.slug_field: kwargs[self.slug_url_kwarg]
         }
-        tround_execution.execute_round(TournamentRound.objects.get(**get_kwargs))
 
-        return result
+        tround = TournamentRound.objects.get(**get_kwargs)
+        threading.Thread(
+            target=lambda: tround_execution.execute_round(tround)).start()
+        messages.success(self.request, 'The round has been started, please wait for email notifications.')
```

Comment:
Ja bym zamienił messages.success() na messages.info(), bo to raczej poinformowanie. (bardzo optional)